### DISCUSSION TITLE
Feature request: Dual List component

### DISCUSSION THREAD

**Discussion by justinsmid** [mantinedev/mantine#365] [Feedback]
Are there any plans for adding a Dual List component?
If not, I would be more than willing to implement one myself if there are no objections.

In terms of design, I would like it to look somewhat like this:

![image](https://user-images.githubusercontent.com/34271675/139653327-77458949-466f-433e-939a-6e589233b23d.png)

**Comment by rtivital** [comment]
Okay, sounds good, feel free to contribute, please follow this guide to get started – https://mantine.dev/pages/contributing/
Use other components as reference, do not forget to add tests. If you have any questions feel free to reach out on Discord in Contributing channel.
